File: 2.0/python/keywords/getKeywordLease.py
from callfire.client import CallfireClient

client = CallfireClient('api-login', 'api-password')
response = client.keywords.getKeywordLease(
    keyword='SUN',
    # return only specific fields
    fields='items(keyword,leaseBegin,leaseEnd)'
).result()

print(response)

File: 2.0/php/keywords/getKeywordLease.php
<?php

class ApiClientSample {

    public static function main() {
        $client = \CallFire\Api\DocumentedClient::createClient("login", "password");
        $request = $client->getKeywordLease();
        $request->getOperationConfig()->setPathParameters(array("keyword" => "SUN"));
        $request->getOperationConfig()->setQueryParameters(array("fields" => "items(keyword,leaseBegin,leaseEnd)"));
        $result = $client->request($request);
        $json = json_decode($result->getBody());
    }
}

ApiClientSample::main();

